api/src/main/scala/JobUtils.scala

```diff
@@ -0,0 +1,140 @@
+package org.wikiwatershed.mmw.geoprocessing
+
+import java.util.concurrent.atomic.LongAdder
+import collection.mutable
+import collection.concurrent.TrieMap
+import spray.json._
+import DefaultJsonProtocol._
+import geotrellis.proj4.{CRS, ConusAlbers, LatLng, WebMercator}
+import geotrellis.raster._
+import geotrellis.raster.rasterize._
+import geotrellis.raster.rasterize.Rasterizer.Options
+import geotrellis.raster.mapalgebra.focal.Kernel
+import geotrellis.vector._
+import geotrellis.vector.io._
+import geotrellis.spark._
+import geotrellis.spark.io._
+import geotrellis.spark.io.s3._
+
+trait JobUtils {
+  val baseLayerReader = S3CollectionLayerReader("datahub-catalogs-us-east-1", "")
```

Comment:
Wonder if this should be made into a parameter, a-la `geoprocessing.port` and `.hostname`. `geoprocessing.s3bucket`?